test/libsolidity/syntaxTests/controlFlow/storageReturn/assembly/switch_fine.sol

```diff
@@ -1,20 +1,14 @@
 contract C {
     struct S { bool f; }
     S s;
-    function f(uint256 a) internal pure returns (S storage c) {
-        assembly {
-            switch a
-            default { c_slot := s_slot }
```

Comment:
Can you keep this? It is useful to know that the check also works for the "only default" case.

Reply:
I have separated it to a new file `test/libsolidity/syntaxTests/controlFlow/storageReturn/assembly/switch_only_default_warn.sol` It bothered me that the name has `_fine` but still issues a warning.